pom.xml

```diff
@@ -196,7 +196,7 @@
             <dependency>
                 <groupId>org.eclipse.jetty</groupId>
                 <artifactId>jetty-servlets</artifactId>
-                <version>${jetty.servlets.version}</version>
+                <version>${jetty.version</version>
```

Comment:
@sumit-confluent please fix the missing bracket 
```suggestion
                <version>${jetty.version}</version>
```

Reply:
thanks for pointing out!! added the missing bracket